src/main/groovy/com/github/roroche/plantuml/urls/CompileClasspathUrlsFactory.groovy

```diff
@@ -0,0 +1,38 @@
+package com.github.roroche.plantuml.urls
+
+import org.gradle.api.Project
+
+import static com.github.roroche.plantuml.android.AndroidProjectType.*
+
+/**
+ * Factory for creating the correct classpath based on the type of a project
+ */
+class CompileClasspathUrlsFactory {
+    static Urls.Wrap createCompileClasspathUrls(Project project) {
```

Comment:
Please use the high-level type, which is `Urls` in this case. The motivation is to always work by contract: https://www.yegor256.com/2014/11/20/seven-virtues-of-good-object.html#2-he-works-by-contracts